test/unit/log.c

```diff
@@ -59,7 +65,7 @@ TEST_BEGIN(test_log_enabled_direct) {
 	assert_d_eq(count, 10, "Mis-logged!");
 
 	count = 0;
-	strcpy(log_var_names, "l1.a");
+  log_var_names[0] =  'l';
```

Comment:
This doesn't feel right. Should we increase the size of `log_var_names` instead?

Reply:
I think this is spurious, since we don't actually run any of this code when logging is disabled.

Reply:
The problem is that when this code is compiled `JEMALLOC_LOG` is not defined, so `JEMALLOC_LOG_VAR_BUFSIZE == 1`.

Reply:
But in those cases, config_log is false, and so the `test_skip_if(!config_log);` above returns early, so that this code doesn't execute. (Or, if not, then that's a true bug that we should fix in the implementation).

Reply:
@davidtgoldblatt maybe these test should then just be `#ifdef`ed out if `JEMALLOC_LOG` is not defined?

Reply:
@davidtgoldblatt an alternative is to leave it as it was and just silence the warning